go.mod

```diff
@@ -21,7 +20,7 @@ require (
 	github.com/golang/protobuf v1.3.3
 	github.com/golangci/golangci-lint v1.21.0 // indirect
 	github.com/google/certificate-transparency-go v1.1.0 // indirect
-	github.com/google/go-cmp v0.3.1
+	github.com/google/go-cmp v0.4.0
```

Comment:
I'm surprised `go mod tidy` has updated a dependency - I thought it just added missing dependencies and pruned unused ones.

Reply:
`go build` and `go mod tidy` can update dependencies when the graph of minimum versions across all dependencies changes.